src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,154 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head = null;
+    private Node<T> tail = null;
+    private int size = 0;
```

Comment:
```suggestion
    private Node<T> head;
    private Node<T> tail;
    private int size;
```

https://docs.oracle.com/javase/tutorial/java/nutsandbolts/datatypes.html